course-mw/sunbird-util/sunbird-platform-core/common-util/src/main/java/org/sunbird/common/models/util/cloud/CloudService.java

```diff
@@ -0,0 +1,20 @@
+package org.sunbird.common.models.util.cloud;
+
+import java.io.File;
+import java.util.List;
+
+/** Created by arvind on 24/8/17. */
+public interface CloudService {
```

Comment:
CloudService and CloudUtils, both the classes are not used any where. Can you delete it.